### DISCUSSION TITLE
How can I operate online and offline products

### DISCUSSION THREAD

**Discussion by mamiao** [saleor/saleor#11471] [Deployment]
hi all：
I have installed the saleor/dashboard/storefront to deploy it in aws EC2, but the salor can only see 
![image](https://user-images.githubusercontent.com/8089170/206994118-e717774d-b63a-4112-935e-562c6f1aad7d.png)
, How can I operate online and offline products? Thanks

Since i have tested the docker enviroment, all of the dashboard, storefront and the saleor are in the port:8000 website

**Comment by karolkielecki** [comment]
Hi @mamiao, could you please tell us what you would like to achieve? What do you mean by "operate offline products"? Also, have you considered using Saleor Cloud? You can open a [free developer account](https://cloud.saleor.io/signup).
